src/main/java/frc/robot/RobotContainer.java

```diff
@@ -8,6 +8,21 @@
 import edu.wpi.first.wpilibj2.command.Command;
 import edu.wpi.first.wpilibj2.command.button.*;
 import frc.robot.commands.*;
+import frc.robot.commands.ClampCmd;
```

Comment:
Are these needed if there's a `import frc.robot.commands.*;` line?

Reply:
does it just import every comand we have in our code?